### ISSUE TITLE
"Could not find implementations (...) jsx-wrap-multiline"

### ISSUE THREAD

**Issue by Kovensky** [palantir/tslint-react#46]
Running tslint with the following minimal configuration fails with `tslint-react@2.1.0` (but works with `2.0.0`):

```json
{
  "extends": [
    "tslint-react"
  ]
}
```

The error message is the following, printed for each of the linted files:

```
Could not find implementations for the following rules specified in the configuration:
    jsx-wrap-multiline
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```

**Comment by GrzegorzKaczan** [comment]
Type in rules filename: jsxWrapMu**lit**lineRule.ts

**Comment by adidahiya** [comment]
Sorry about that; I've released v2.1.1 with the fix. Building on Circle CI now...

**Comment by adidahiya** [comment]
v2.1.1 is now published to NPM.
